mesh/profiles/views.py

```diff
@@ -26,20 +29,26 @@ def get(self, request, account_id, *args, **kwargs):
             return JsonResponse({'error': 'Invalid request. Account does not exist'}, status=404)
         except Profile.DoesNotExist:
             return JsonResponse({'error': 'Invalid request. Profile does not exist'}, status=404)
+      
+    def post(self, request, account_id, *args, **kwargs):
+        """
+        Handle POST requests.
 
-    def post(self, request, *args, **kwargs):
-        if request.method == "POST":
-            data = request.POST
-            if 'accountID' not in data:
-                return JsonResponse({'error': 'Invalid request. Missing accountID field.'}, status=401)
-            if 'biography' not in data:
-                return JsonResponse({'error': 'Invalid request. Missing biography field.'}, status=400)
-            else:
-                print(data.get('accountID') + " " + data.get('biography'))  # TODO: Save to database
-                return JsonResponse({'message': 'biography saved successfully'}, status=200)
-        else:
-            return JsonResponse({'error': request.method + ' Method not allowed'}, status=405)
+        Updates the biography of the listed profile.
 
+        Returns a JSON response containing the saved biography of the specified profile through id.
+        """
+        data = json.loads(request.body)
+
+        if 'accountID' not in data:
+            return JsonResponse({'error': 'Invalid request. Missing accountID field.'}, status=401)
```

Comment:
There is a folder in 'mesh' for the exceptions that we have. This doesn't look like this was your code but maybe we can open up a task for tech debt or you can take it on a part of this task to properly use the exceptions for invalid input.  I think I am leaning towards another tech debt task.

Reply:
Yeah, that sounds like a tech-debt task 👍